src/dataaccess/.gitignore

```diff
@@ -0,0 +1,15 @@
+/Auth.class
```

Comment:
remove this file also... for the same reason you will remove the other `.gitignore` file